### PR TITLE
GitHub action to build PPG docs with mike for deployment on Netlify

### PR DESCRIPTION
**Pull request by nastena1606**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+name: Build Percona Distribution PostgreSQL 13 docs
```
Comment (ademidoff) [inline .github/workflows/main.yml:1]: Do we want to have this action for v13 of the docs only, or for every other subsequent action?

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+    steps: 
+      #Pull the latest changes
+      - name: Chekout code
+        uses: percona-platform/checkout@v2
+        with:
+          fetch-depth: 0
+      #Prepare the env
+      - name: Set up Python
+        uses: actions/setup-python@v2
```
Comment (ademidoff) [inline .github/workflows/main.yml]:
```suggestion
        uses: percona-platform/setup-python@v2
```

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+      - name: Setup PlantUML dependencies
+        run: |
+          sudo apt install -y graphviz 
+     
+     # Deploy docs
+      - name: Deploy docs
+        run: |
+          mike deploy 13 -F mkdocs-netlify.yml -b netlify -p
+          mike set-default 13 -b netlify -p
+          mike retitle 13 "13 (LATEST)" -b netlify -p
```
Comment (ademidoff) [inline .github/workflows/main.yml]: add an empty line (best practice)

File: .github/workflows/main.yml (hunk 1 above)
Comment (nastena1606) [reply .github/workflows/main.yml:1]: For every active version which is 13, 12, 11

Comment (ademidoff) [reply .github/workflows/main.yml:1]: Then we may have to put multiple values to the `branches` value right below )

**Review by ademidoff — APPROVED** [review]

**Pull request merged**